pcs/instance_manager.py

```diff
@@ -27,20 +26,24 @@ def __init__(
         self.register_attributes(["instance_of", "argument_set"])
         self._instance = None
 
-    def get_object(self):
+    def get_new_object(self):
         if self._instance:
             return self._instance
```

Comment:
Does it make sense to return the cached `_instance` in `get_new_object()`?  From the name of the method, I'd guess it'd give me a new object even if there's a cached instance.